### ISSUE TITLE
Snowfall weather limits sight to about 10 meters

### ISSUE THREAD

**Issue by cmdflow** [altmp/altv-issues#1242]
### Description of the problem

player.setWeather(10) results in this:

https://media.discordapp.net/attachments/826852441080463400/916818775226056725/unknown.png?width=705&height=444

Post processing is set to normal. No grafic mods installed. reported by multiple users.

### Reproduction steps

set player.setWeather(10)

### Expected behaviour

should not be that less view distance?

### Operating system

Debian 10 Linux version 4.19.0-18-amd64 

### Version

10

**Comment by cmdflow** [comment]
https://i.imgur.com/OTnua1D.jpg

**Comment by C0kkie** [comment]
I dont think that alt:V is affecting the weather effects, pretty sure it has something todo with gta

**Comment by lokuPadoku** [comment]
I can remember having this issue if I had Mod installed like Redux and blizzard kicks in.
